### PR TITLE
Adjust runbooks url to new path

### PR DESCRIPTION
**Pull request by RiRa12621**
The base path for runbooks in github.com/openshift/runbooks has been adjusted, this adjusts the path in `runbooks_url` accordingly

### CODE REVIEW

**Comment by RiRa12621** [comment]
/assign @hexfusion
doing as the bot told me to

File: manifests/0000_90_etcd-operator_03_prometheusrule.yaml [hunk 1]
```diff
@@ -132,10 +119,7 @@ spec:
     rules:
     - alert: etcdHighNumberOfLeaderChanges
       annotations:
-        description: 'etcd cluster "{{ $labels.job }}": {{ $value }} leader changes
-          within the last 15 minutes. Frequent elections may be a sign of insufficient
-          resources, high network latency, or disruptions by other components and
-          should be investigated.'
+        description: 'etcd cluster "{{ $labels.job }}": {{ $value }} leader changes within the last 15 minutes. Frequent elections may be a sign of insufficient resources, high network latency, or disruptions by other components and should be investigated.'
```
Comment (lilic) [inline manifests/0000_90_etcd-operator_03_prometheusrule.yaml:122]: Something wrong with your formating or why is this change? 🤔

**Review by lilic — COMMENTED** [review]
Thanks, one question

**Review by lilic — APPROVED** [review]
/lgtm

Its due to https://github.com/brancz/gojsontoyaml/pull/8/files. If we want to bring those line breaks back, we need to format it in the mixin itself.

Thanks!

**Comment by lilic** [comment]
/approve

🤷‍♀️

File: manifests/0000_90_etcd-operator_03_prometheusrule.yaml (hunk 1 above)
Comment (RiRa12621) [reply manifests/0000_90_etcd-operator_03_prometheusrule.yaml:122]: https://github.com/openshift/cluster-etcd-operator/pull/632#pullrequestreview-713546641

**Pull request merged**